x-pack/legacy/plugins/siem/public/pages/case/components/bulk_actions/index.tsx

```diff
@@ -9,46 +9,51 @@ import { EuiContextMenuItem } from '@elastic/eui';
 import * as i18n from './translations';
 
 interface GetBulkItems {
+  caseStatus: string;
   closePopover: () => void;
   deleteCasesAction: (cases: string[]) => void;
   selectedCaseIds: string[];
-  caseStatus: string;
+  updateCaseStatus: (status: string) => void;
 }
 
 export const getBulkItems = ({
-  deleteCasesAction,
-  closePopover,
   caseStatus,
+  closePopover,
+  deleteCasesAction,
   selectedCaseIds,
+  updateCaseStatus,
 }: GetBulkItems) => {
   return [
     caseStatus === 'open' ? (
```

Comment:
I think it would be nice if we have a type of all the caseStatus defined.

Reply:
yes i agree. i can do this in a follow up PR, i have another one up that deals a lot with status right now

Reply:
https://github.com/elastic/kibana/issues/60575